crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -259,6 +260,36 @@ public Response bulk(String request) {
         return Response.status(st.getHttpStatus()).entity(st.toString()).build();
     }
 
+    /**
+     * GET /generate/<entity>/<version>/<field>?n=<n>
+     *
+     * @param entity name of the entity
+     * @param version Entity version
+     * @param field Name of the field in the entity containing the generator
+     * @param n Number of values to be generated
+     *
+     * @return A lightblue response, with "processed" containing an array of generated values.
+     */
+    @GET
+    @LZF
+    @Path("/generate/{entity}/{version}/{field}")
+    public Response generate(@PathParam("entity") String entity,
+                             @PathParam("version") String version,
+                             @PathParam("field") String field,
```

Comment:
Changed that to path

On Wed, May 4, 2016 at 12:39 PM, Naveen Malik <notifications@github.com>
wrote:

> In
> crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java
> <https://github.com/lightblue-platform/lightblue-rest/pull/229#discussion_r62091817>
> :
>
> > +    /**
> > +     * GET /generate/<entity>/<version>/<field>?n=<n>
> > +     *
> > +     * @param entity name of the entity
> > +     * @param version Entity version
> > +     * @param field Name of the field in the entity containing the generator
> > +     * @param n Number of values to be generated
> > +     *
> > +     * @return A lightblue response, with "processed" containing an array of generated values.
> > +     */
> > +    @GET
> > +    @LZF
> > +    @Path("/generate/{entity}/{version}/{field}")
> > +    public Response generate(@PathParam("entity") String entity,
> > +                             @PathParam("version") String version,
> > +                             @PathParam("field") String field,
>
> Is it better to call this path instead of field?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-rest/pull/229/files/0bc69428bb58e29e04c0745b8fdbf722b633a7bc#r62091817>
>